pkg/util/util.go

```diff
@@ -55,3 +56,9 @@ func (dt *ThroughputCalc) Start() {
 		}()
 	}
 }
+
+// MaskPassword is used to mask sensitive password data before outputing to persistent stream like logs.
+func MaskPassword(s string) string {
+	s = regexp.MustCompile(`password=(\s*?)'([^']+?)'`).ReplaceAllString(s, "password=$1'****'")
```

Comment:
should we save these Regexp objects (returned by MustCompile) as global vars or const so that we don't need to recompile on every call to the function?